### ISSUE TITLE
Analyze workflow automation in RDPC DEV and QA

### ISSUE THREAD

**Issue by UmmulkiramR** [icgc-argo/workflow-graph-node#95]

**Comment by UmmulkiramR** [comment]
Automation ran successfully on DEV. Next would be to get is started on QA.

**Comment by UmmulkiramR** [comment]
Automation Stats for DEV and QA with HT on

**DEV:**
The cycle on Dev successfully executed from start to end in about 4 and 1/2 hours. 
Alignment -> Mutect/SangerWXS --> Open Access(ran twice)
CPU and Mem requested: 
     bwaMemAligner -> 24CPUs and 128 mem 
     sangerWxsVariantCaller -> 24CPUs and 128 mem

**QA:**
The cycle on QA successfully executed from start to end in about 2 hours and 40 mins. This was with a smaller dataset.
Alignment -> Mutect/SangerWXS --> Open Access(ran 4 times)
CPU and Mem requested: 
     bwaMemAligner -> 24CPUs and 128 mem 
     sangerWxsVariantCaller -> 16CPUs and 80 mem

**Comment by Buwujiu** [comment]
UK is working on the load testing in qa, will close it once testing is complete.

**Comment by UmmulkiramR** [comment]
Load testing automation in QA encountered an issue where one of the downstream jobs(open access filtering) in the pipeline did not execute. Initial trouble shooting and analysis of the console logs revealed that somewhere during the execution the message id in the payload from queue got replaced with the message id from another message in the queue which caused that other message to get rejected. The rejected message belonged to the job that did not execute.

Why this is happening is still unclear and needs more investigation.

**Comment by Buwujiu** [comment]
logged a new ticket for QA issue, closing the current ticket.
